src/styles/components/_form.scss

```diff
@@ -52,6 +52,12 @@
     padding: rem(4px 12px);
   }
 
+  &__recaptcha {
+    display: flex;
+    justify-content: center;
+    margin-bottom: rem(24px);
+  }
```

Comment:
You can combine `margin-bottom` with the above `&:not(:last-child)` and for the `display` properties combine them with `&__actionable`.